src/test/kotlin/org/jetbrains/intellij/IntelliJPluginSpec.kt

```diff
@@ -1,14 +1,13 @@
 package org.jetbrains.intellij
 
 import org.gradle.api.plugins.JavaPlugin
+import org.gradle.testkit.runner.BuildResult
```

Comment:
[Applies to entire file]
I cleaned the tests:
- reduced code duplication
- reduces code noise in testing methods, so they include actual tested context and assertions
- added messages to assertions - before there was a lot of "expected true, but got false" messages and now there will be more precise

All existing tests and logic should be preserved.